docs-ref-services/storage-blob-readme.md

```diff
@@ -34,7 +34,8 @@ Blob storage is ideal for:
 [Azure storage account](https://docs.microsoft.com/azure/storage/common/storage-account-overview) to use this package.
 
 ### Install the package
-Install the Azure Storage Blobs client library for Python with [pip](https://pypi.org/project/pip/):
+Install the Azure Storage Blobs client library for Python - Version 12.4.0 
```

Comment:
is it as expected?

Reply:
Aha! NO!  This is exactly what I was looking for when asking folks to review!  Automation going insane 👍